config/coreboot-p8z77-m_pro-tpm1.config

```diff
@@ -0,0 +1,17 @@
+CONFIG_USE_BLOBS=y
+CONFIG_VENDOR_ASUS=y
+CONFIG_CBFS_SIZE=0x7E7000
+CONFIG_BOARD_ASUS_P8Z77_M_PRO=y
+CONFIG_HAVE_IFD_BIN=y
+CONFIG_HAVE_ME_BIN=y
+CONFIG_IFD_BIN_PATH="@BLOB_DIR@/p8z77-m_pro/ifd.bin"
+CONFIG_ME_BIN_PATH="@BLOB_DIR@/p8z77-m_pro/me.bin"
+CONFIG_NO_GFX_INIT=y
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_TPM1=y
+CONFIG_DRIVERS_PS2_KEYBOARD=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_8=y
+CONFIG_PAYLOAD_LINUX=y
+CONFIG_PAYLOAD_FILE="@BOARD_BUILD_DIR@/bzImage"
+CONFIG_LINUX_INITRD="@BOARD_BUILD_DIR@/initrd.cpio.xz"
+CONFIG_LINUX_COMMAND_LINE="intel_iommu=on intel_iommu=igfx_off nohz=off"
```

Comment:
it bugged the hell out of me since the early first desktop, but just an annoyance.  similar with crng init - this board decides to hit that point after enough keystrokes of entropy and that happens right at the point of typing in the custom TPM password, so theres a console output in the middle of the password echo.